addons/event/models/event.py

```diff
@@ -431,6 +444,15 @@ class EventRegistration(models.Model):
         ('open', 'Confirmed'), ('done', 'Attended')],
         string='Status', default='draft', readonly=True, copy=False, tracking=True)
 
+    is_paid = fields.Boolean(string='Is paid')
```

Comment:
I don't understand the purpose of a "is_paid" field, to set manually, and not linked to any sale order, moreover in base event module. If people want to manually manage paid events without event_sale they can use odoo studio and add fields manually.

Reply:
@tde-banana-odoo 
This logic of this field is in the task "Event phase 2" ;)